sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -105,6 +118,7 @@ private Analytics() {
         mFactories.put(StartSessionLog.TYPE, new StartSessionLogFactory());
         mFactories.put(PageLog.TYPE, new PageLogFactory());
         mFactories.put(EventLog.TYPE, new EventLogFactory());
+        sTransmissionTargets = new HashMap<>();
```

Comment:
why static? we use singleton and getInstance().doSomething() everywhere. Only static field is usually the instance itself.